seismic/xcorqc/correlator.py

```diff
@@ -65,7 +43,7 @@ def __init__(self, asdf_file_name, netsta_list='*'):
         self.metadata = defaultdict(list)
 
         rtps = []
-        for netsta in self.fds.unique_coordinates.keys():
+        for netsta in list(self.fds.unique_coordinates.keys()):
```

Comment:
In Python 2 and 3, to iterate over keys only you can write
`  for netsta in self.fds.unique_coordinates:`
which may be more efficient due to use of generator expressions.

Reply:
When I run 2to3, it complains and suggests keys() should be explicitly converted to a list.

```
-        for netsta in self.fds.unique_coordinates.keys():
+        for netsta in list(self.fds.unique_coordinates.keys()):
```

Reply:
It won't complain if you remove the `.keys()`, just `for netsta in self.fds.unique_coordinates:`